src/components/Input/Input.stories.js

```diff
@@ -1,16 +1,15 @@
 import Input from './Input'
 
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
If not necessary, you could delete the comment...

Reply:
This is an autmatic comment from eslint, only because I have disabled eslint for the following 3 lines. Otherwise I always get a problem message which is annoying because it is only a simple warning ;-) I don't even know how to deactivate the lines without this comment appearing here. Do you have any idea?